Vagrantfile

```diff
@@ -100,6 +100,26 @@ Vagrant.configure("2") do |config|
     centos.vm.synced_folder ".", "/vagrant", type: "rsync"
   end
 
+  config.vm.define "x86_64-c10s" do |centos|
+    centos.vm.box = "centos-10-stream"
+    centos.vm.box_url = get_box("https://cloud.centos.org/centos/10-stream/x86_64/images/", /.*latest\.x86_64\.vagrant-libvirt\.box$/)
+
+    centos.vm.provision "shell", inline: <<-SHELL
+       dnf config-manager --set-enabled crb
+    SHELL
+    centos.vm.provision "common", type: "shell", inline: $bootstrap_rhel_common
+    # Repo https://mirror.stream.centos.org/SIGs/10-stream/nfv/x86_64/openvswitch-2/
+    # exists but at the moment does not provide the selinux dependency that's
+    # required to install openvswitch. Use a COPR repo instead.
```

Comment:
Is there a link to follow the issue somewhere, so we know when to remove this workaround?

Reply:
done